qartod/adcp/adcp_qartod_gross_range_test_values.csv

```diff
@@ -661,7 +661,7 @@ CP01CNPM,RII01,02-ADCPTG010,adcp_engineering,{'inp': 'speed_of_sound'},"{'qartod
 CP01CNPM,RII01,02-ADCPTG010,adcp_engineering,{'inp': 'adc_transmit_energy'},"{'qartod': {'gross_range_test': {'fail_span': [0, 10]}}}",Derived from vendor documentat and instrument configuration,Assumes a minimum voltage of 20 VDC (counts = round(20 * 1000000 / 2092719))
 CP01CNPM,RII01,02-ADCPTG010,adcp_velocity_earth,{'inp': 'pitch'},"{'qartod': {'gross_range_test': {'suspect_span': [-1500, 1500], 'fail_span': [-2000, 2000]}}}",Derived from vendor documentation,Valid range is listed as +/- 20 degrees in the manual with tilts greater than +/- 15 degrees as suspect
 CP01CNPM,RII01,02-ADCPTG010,adcp_velocity_earth,{'inp': 'roll'},"{'qartod': {'gross_range_test': {'suspect_span': [-1500, 1500], 'fail_span': [-2000, 2000]}}}",Derived from vendor documentation,Valid range is listed as +/- 20 degrees in the manual with tilts greater than +/- 15 degrees as suspect
-CP01CNPM,RII01,02-ADCPTG010,adcp_velocity_earth,{'inp': 'temperature'},"{'qartod': {'gross_range_test': {'fail_span': [-5, 45]}}}",Derived from vendor documentation,"Standard temperature sensor range is listed as -5 to 45 in the specifications. Not a standard IOOS QARTOD parameter, using a proxy for speed of sound in the adcp_engineering stream"
+CP01CNPM,RII01,02-ADCPTG010,adcp_velocity_earth,{'inp': 'temperature'},"{'qartod': {'gross_range_test': {'fail_span': [-500, 4500]}}}",Derived from vendor documentation,"Standard temperature sensor range is listed as -5 to 45 in the specifications. Not a standard IOOS QARTOD parameter, using a proxy for speed of sound in the adcp_engineering stream"
```

Comment:
Checked that all "temp" and "temperature" parameters have correct range for cdeg_C units, looks good